packages/lucide-svelte/scripts/appendLicense.mjs

```diff
@@ -32,18 +27,13 @@ for (const file of files) {
     license = getJSBanner();
   }
 
-  if (/\.svelte/.test(ext)) {
-    license = getSvelteBanner();
-  }
-
   if (license) {
     newContents = license + contents;
-
   }
 
   // Places icon block comment at the top of the Svelte component class
-  if(/icons\/(.*?)\.svelte\.d\.ts/.test(filepath)) {
-    const svelteFilepath = filepath.replace('.d.ts', '')
+  if (/icons\/(.*?)\.svelte\.d\.ts/.test(filepath)) {
+    const svelteFilepath = filepath.replace('.d.ts', '');
     let svelteFileContents = await readFile(svelteFilepath, { encoding: 'utf-8' });
 
     const blockCommentRegex = /\/\*\*[\s\S]*?\*\//;
```

Comment:
We need to change this regex, this now also matches with the license block comment which results in twice adding the license block comment. 

Here is the correct new regex:
``` suggestion
    const blockCommentRegex = /\/\*\*\n\s\*\s(@component\s@name)[\s\S]*?\*\//;
```